.expeditor/verify.pipeline.yml

```diff
@@ -84,8 +79,8 @@ steps:
         privileged: true
         image: "chefes/a1-buildkite"
         environment:
-            - OCTOKIT_ACCESS_TOKEN
-            - COMPONENT=src/oc_erchef
+          - OCTOKIT_ACCESS_TOKEN
```

Comment:
Do we really need `OCTOKIT_ACCESS_TOKEN` in all these tests? to my knowledge that env var is specific to the octokit ruby library.